biit_server/account_handler.py

```diff
@@ -30,8 +40,17 @@ def account_post(request):
 
 
 def account_get(request):
-    """
-    Handles the account GET endpoint
+    """Handles the account GET endpoint
+    Validates data sent in a request then calls the database to get the row of the associated account
+
+    Args:
+        request: A request object that contains args with keys: email
+
+    Returns:
+        Http 200 string response with the associated account information
```

Comment:
can we return a json here too? that would make sense since this is the `GET` method